src/cpp/DQM/render.cc

```diff
@@ -646,9 +646,11 @@ class VisDQMImageServer : public DQMImplNet<VisDQMObject>
 public:
   static const uint32_t DQM_MSG_GET_IMAGE_DATA	= 4;
   static const uint32_t DQM_MSG_DUMP_PROFILE	= 5;
-  static const uint32_t DQM_MSG_GET_JSON_DATA	= 6;
+  static const uint32_t DQM_MSG_GET_JSON_DATA   = 6;
+  static const uint32_t DQM_MSG_GET_JSROOT_DATA	= 7;
```

Comment:
Not sure why github show indentation here. checked  code on sublime, geddit, nano editors all of them show code alighted. 
![image](https://cloud.githubusercontent.com/assets/11797743/24351214/2697b484-12e5-11e7-847e-e0457dc8facd.png)